### PR TITLE
Update extension badge text when LH is running

### PR DESCRIPTION
**Pull request by ebidel**
R: @paulirish @wardpeet all

This expands on #1185 by updating the tooltip and badge text when LH is testing a page. Makes it more obvious something is going on if the popup has been dismissed.

screen shot 2016-12-30 at 3 20 46 pm
screen shot 2016-12-30 at 3 20 52 pm

### CODE REVIEW

File: lighthouse-extension/app/src/lighthouse-background.js [hunk 1]
```diff
@@ -90,17 +109,16 @@ window.runLighthouseForConnection = function(connection, url, options, requested
   // Add url and config to fresh options object.
   const runOptions = Object.assign({}, options, {url, config});
 
-  lighthouseIsRunning = true;
```
Comment (paulirish) [inline lighthouse-extension/app/src/lighthouse-background.js]: I'd prefer to keep the explicit `lighthouseIsRunning = true;` inline here and below but call a `updateBadgeUI()` immediately after each.

Comment (ebidel) [inline lighthouse-extension/app/src/lighthouse-background.js]: Done.

File: lighthouse-extension/app/src/lighthouse-background.js [hunk 2]
```diff
@@ -74,6 +74,23 @@ function filterConfig(config, requestedAggregations) {
...
 
 /**
+ * Sets the extension badge text.
+ * @param {string=} optUrl If present, sets the badge text to "Testing <url>".
+ *     Otherwise, restore the default badge text.
+ */
+function updateBadgeUI(optUrl) {
+  if (window.chrome && chrome.runtime) {
+    const manifest = chrome.runtime.getManifest();
+    const title = optUrl ? `Testing ${optUrl}` : manifest.browser_action.default_title;
```
Comment (paulirish) [inline lighthouse-extension/app/src/lighthouse-background.js]: now we can use `lighthouseIsRunning` for these conditionals. that work for you?

Comment (ebidel) [inline lighthouse-extension/app/src/lighthouse-background.js]: done

**Comment by wardpeet** [comment]
thanks for merging, sorry wasn't available:)

**Pull request merged**